lib/usdUfe/ufe/UsdUIInfoHandler.cpp

```diff
@@ -159,15 +114,10 @@ bool UsdUIInfoHandler::treeViewCellInfo(const Ufe::SceneItem::Ptr& item, Ufe::Ce
     return changed;
 }
 
-Ufe::UIInfoHandler::Icon UsdUIInfoHandler::treeViewIcon(const Ufe::SceneItem::Ptr& item) const
+UsdUIInfoHandler::SupportedTypesMap UsdUIInfoHandler::getSupportedIconTypes() const
```

Comment:
New virtual method to return the list of supported icon types. Can be overridden by derived class to add extra icons (such as from the MayaUsdUIInfoHandler).